config/resource_suppressions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+    "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+    
+      <!-- Suppressions inside test and it folders -->
+      <suppress checks="NewlineAtEndOfFile" files="(src[\\/]it[\\/].*rule731selfexplanatory[\\/]InputJavadocMethodCheck\.java|src[\\/]test[\\/].*Newline.*AtEndOfFile\.java|src[\\/]test[\\/].*astprinter[\\/]InputJavadocComment\.javadoc|src[\\/]test[\\/].*astprinter[\\/]InputJavadocWithError\.javadoc|src[\\/]test[\\/].*checks[\\/]InputEmptyFile\.txt|src[\\/]test[\\/].*grammars[\\/]javadoc[\\/]htmlTags[\\/].*|src[\\/]test[\\/].*grammars[\\/]javadoc[\\/]javadocTags[\\/].*|src[\\/]test[\\/].*grammars[\\/]InputRegressionEmptyAst\.txt|src[\\/]test[\\/].*grammars[\\/]javadoc[\\/]InputLeadingAsterisks\.txt|src[\\/]test[\\/].*grammars[\\/]comments[\\/]InputFullOfSinglelineComments.java|src[\\/]test[\\/].*grammars[\\/]InputEmbeddedNullChar\.java)"/>
```

Comment:
This is too hard to read and maintain as one line string. What if we accidentally screw expression somewhere.
Please split it up. I am fine with one file/directory per line.